src/early_multicellular_stage/editor/ReproductionOrder.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Godot;
+
+/// <summary>
+///   Handles showing and changing the order in which cells in an early multicellular creature will divide.
+/// </summary>
+public class ReproductionOrder : MarginContainer
+{
+#pragma warning disable CA2213
```

Comment:
This warning suppress should not be around the NodePaths. Also the first nodepath should be nullable to detect when this class was instanced in the editor. You'll need to also dispose the nodepaths. Please have a look at our other GUI control types to see how they should be structured.